web-api/openapi/front_office.yaml

```
@@ -1,20 +1,21 @@
 openapi: 3.1.0
+
 info:
   title: Front Office API
   version: 2.3.0
   description: |-
     # Front Office
     The front office is typically used within front-end apps, for example a website or a mobile application.
-    With this API, you can handle interactions with documents and request a list of personalized documents.
+    With this API, you can handle interactions with documents and search for personalized documents.
 
     ## Format
-    All the request bodies in this API are JSON encoded and their `Content-Type` headers should be set to `application/json`.
+    All the request bodies in this API are JSON encoded and their `content-type` header should be set to `application/json`.
```

:art: Http headers are generally referenced in their capitalized form if you look at RFCs and similar.

```suggestion
    All the request bodies in this API are JSON encoded and their `Content-Type` header should be set to `application/json`.
```

---

the issue is that upper case characters in header names are invalid in http/2 and http/3, see for example https://www.rfc-editor.org/rfc/rfc7540#section-8.1.2 for http/2. even though we shouldn't rule out http/1, we should still stick to the conventions of the newer versions.